src/lib.rs

```diff
@@ -104,6 +109,27 @@ mod reexport {
 #[plugin_registrar]
 #[cfg_attr(rustfmt, rustfmt_skip)]
 pub fn plugin_registrar(reg: &mut Registry) {
+    let conferr = match conf::conf_file(reg.args()) {
+        Ok(Some(file_name)) => {
+            conf::read_conf(&file_name, true)
+        }
+        Ok(None) => {
+            conf::read_conf("Clippy.toml", false)
+        }
+        Err((err, span)) => {
+            reg.sess.struct_span_err(span, err).emit();
+            return;
```

Comment:
We should not consider this a fatal error. We should just continue with default options if the conf file is broken.